bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dPlotSquaredPlot.java

```diff
@@ -76,6 +78,14 @@ public dPlotSquaredPlot setPrefix(String prefix) {
     }
 
     @Override
+    public boolean equals(Object a) {
+        if (a instanceof dPlotSquaredPlot) {
+            return ((dPlotSquaredPlot) a).plot.getId().equals(plot.getId());
+        }
+        return false;
+    }
```

Comment:
You shouldn't override the `equals` method in Java without also overriding `hashCode`, it leads to some funky stuff. Let us know if you need help figuring that out

Reply:
I tried contacting you on Discord about this. I do need some help with this one.